mmv1/templates/terraform/examples/network_connectivity_policy_basic.tf.erb

```diff
@@ -0,0 +1,34 @@
+resource "google_compute_network" "consumer_net" {
+  name                    = "<%= ctx[:vars]['consumer_network_name'] %>"
+  auto_create_subnetworks = false
+}
+
+resource "google_compute_subnetwork" "consumer_subnet" {
```

Comment:
Just to verify, I don't see the consumer network/subnetwork used elsewhere in this test (same with the update test). Is that intended?

Reply:
That's right, I was just adding that because this resource is part of one product that brings this concept of connecting two different VPCs, a consumer side and a producer side. And it's a basic scenario, I can remove that.
I just keep that in here, because we can grow this basic example to a more complex one.

Reply:
yeah, makes sense. Done that.